src/azul/plugins/metadata/hca/aggregate.py

```diff
@@ -24,11 +28,41 @@
     SumAccumulator,
     UniqueValueCountAccumulator,
 )
+from azul.indexer.document import (
+    Aggregate,
+    FieldTypes,
+    pass_thru_int,
+)
 from azul.types import (
     JSON,
 )
 
 
+@dataclass
+class HCAAggregate(Aggregate):
+    total_estimated_cells: int = field(init=False)
+
+    def __post_init__(self):
+        super().__post_init__()
+        self.total_estimated_cells = self._total_estimated_cells()
```

Comment:
Seems overly complicated. Can't you use our default mechanism for lazily computed values: @cached_property?